roles/docker/templates/docker-options.conf.j2

```diff
@@ -1,2 +1,2 @@
 [Service]
-Environment="DOCKER_OPTS={% if docker_options is defined %}{{ docker_options }}{% endif %}"
\ No newline at end of file
+Environment="DOCKER_OPTS={% if docker_options is defined %}{{ docker_options }}{% endif %} --iptables={% if kube_network_plugin == 'flannel' %}false{% else %}true{% endif %}"
```

Comment:
Here if `kube_network_plugin == 'flannel'` you set `--iptables=false`, I think expected the opposite behavior.